content/05-more/08-releases.mdx

```diff
@@ -12,6 +12,42 @@ This page explains the release process of Prisma, how it's versioned and how to
 
 Prisma releases typically happen every two weeks. Note that this is _not_ a hard rule, releases might be postponed for internal reasons.
 
+## Product maturity levels
+
+A release can inclue products or features at different maturity levels. Maturity level describes a product or feature's completeness, and what users can expect in terms of breaking changes.
+
+### Experiment
+
+If a feature or product is an **experimental**:
+
+- We want to test whether a problem genuinely exists and if a proposed prototype would solve it.
+- We are not yet clear about whether we want to add it to the product surface.
+
+### Early Access
+
+If a feature or product is **early access**:
+
+- We have validated a problem and are considering a solution to it, but are not certain whether that solution is complete or a perfect fit.
+- We want to gather more feedback and adjust the solution as necessary, knowing that users are prepared for significant breaking changes
+
+### Preview
+
+If a feature or product is a **preview**:
+
+- We have validated the feature or product in terms of direction and surface.
+- Users can count on the feature or product and associated API to be stable, unless stated otherwise in the release notes and documentation.
```

Comment:
```suggestion
- Users can count on the feature or product and associated API to be mostly stable, unless stated otherwise in the release notes and documentation.
```